compass/ocean/tests/drying_slope/initial_state.py

```diff
@@ -46,42 +39,96 @@ def run(self):
         """
         config = self.config
         logger = self.logger
+        resolution = self.resolution
 
+        # Fetch config options
         section = config['vertical_grid']
-        coord_type = self.coord_type
         thin_film_thickness = section.getfloat('thin_film_thickness') + 1.0e-9
```

Comment:
Is the `1.0e-9` needed here? Maybe we can define a variable with a comment so its less of a magic number.